infrastructure/src/main/java/org/corfudb/infrastructure/CorfuServer.java

```diff
@@ -95,7 +95,7 @@
                     + " -d <level>, --log-level=<level>         Set the logging level, valid levels are: \n"
                     + "                                         ERROR,WARN,INFO,DEBUG,TRACE [default: INFO].\n"
                     + " -Q, --quickcheck-test-mode              Run in QuickCheck test mode\n"
-                    + " -P <seconds>, --cm-poll-interval        Configuration manager poll interval [default: 1]\n"
+                    + " -M <address>:<port>, --mgmt=<address>:<port>     Layout endpoint to seed Management Server\n"
```

Comment:
this should probably be --management-server to make it clear what it is, if the user wants to abbreviate they can use --management.